travis/test_pylint

```diff
@@ -6,19 +6,54 @@ Script to process pylint run and exit with result.
 """
 
 import os
+import ConfigParser
 
 import run_pylint
 import travis_helpers
 
 from getaddons import get_modules_changed
 
+
+def get_extra_params(odoo_version):
+    '''
+    Get extra pylint params by odoo version
+    Transform a seudo-pylint-conf to params,
```

Comment:
This method allow it use a seudo-inherit of configuration file.
To avoid have number-of-odoo-versions X 2 config files (normal and pr conf).
Example:
 - pylint_master.conf
 - pylint_master_pr.conf
 - pylint_90.conf
 - pylint_90_pr.conf
 - pylint_80.conf
 - pylint_80_pr.conf
 - pylint_70.conf
 - pylint_70_pr.conf
 - pylint_61.conf
 - pylint_61_pr.conf
 - ... and new furture versions.
If you need add a new conventions in all versions you will need change all files.

With this method you can use:
  - pylint_lastest.conf
  - pylint_lastest_pr.conf
  - pylint_disabling_70.conf <- disable and overwrite params from pylint lastest
  - pylint_disabling_61.conf <- disable and overwrite params from pylint lastest
If you need add a new conventions in all versions you will need change just in pylint_lastest_pr.conf or pylint_lastest.conf, similar to inherit.